season/Season.js

```diff
@@ -1,43 +1,24 @@
-import ProductSeason from '@/classes-shared/season/ProductSeason'
-
 export default class Season {
-  constructor(singleSeason, destinationInstance = null) {
-    this.id = singleSeason.id
-    this.identifier = singleSeason.identifier
-    this.from = new Date(singleSeason.from)
-    this.to = new Date(singleSeason.to)
-    this.name = singleSeason.name
-    this.createdAt = new Date(singleSeason.createdAt)
-    this.updatedAt = new Date(singleSeason.updatedAt)
-    this.suppSeasonalities = singleSeason.suppSeasonalities
-    this.destinationInstance = destinationInstance
-    this.extAxessTarifBlattGueltCreated =
-      singleSeason.extAxessTarifBlattGueltCreated
-    this.productSeasons = []
-    // parse product season links
-    if (singleSeason.productSeason)
-      this.parseProductSeasons(singleSeason.productSeason)
-  }
-
-  parseProductSeasons(rawProductSeasons) {
-    for (let rawProductSeason of rawProductSeasons) {
-      this.productSeasons.push(new ProductSeason(rawProductSeason))
-    }
+  constructor(params) {
+    this.id = params && params.id ? params.id : null
+    this.from = params && params.from ? new Date(params.from) : null
+    this.to = params && params.to ? new Date(params.to) : null
+    this.name = params && params.name ? params.name : null
```

Comment:
`params.name || null`
For the others too.

Reply:
Only for id and name right. or how would it work with the new Date()?

Reply:
Correct.